hypervisor/dm/vpci/vpci.c

```diff
@@ -483,10 +483,23 @@ static int32_t write_cfg(struct acrn_vpci *vpci, union pci_bdf bdf,
 }
 
 /**
+ * @brief Initialize a vdev structure.
+ *
+ * The function vpci_init_vdev is used to initialize a vdev structure with a PCI device configuration(dev_config)
+ * on a specified vPCI bus(vpci). If the function vpci_init_vdev initializes a SRIOV Virtual Function(VF) vdev structure,
+ * the parameter pf_vdev is the VF associated Physical Function(PF) vdev structure, otherwise the parameter pf_vdev is NULL.
+ * The caller of the function vpci_init_vdev should guarantee execution atomically.
+ *
+ * @param vpci       Pointer to a vpci data structure
+ * @param dev_config Pointer to a dev_config structure
+ * @param pf_vdev    Pointer to a SRIOV PF vdev structure
+ *
  * @pre vpci != NULL
  * @pre vpci.pci_vdev_cnt <= CONFIG_MAX_PCI_DEV_NUM
+ *
+ * @return If there's a successfully initialized vdev structure return it, otherwise return NULL;
  */
-static struct pci_vdev *vpci_init_vdev(struct acrn_vpci *vpci, struct acrn_vm_pci_dev_config *dev_config)
+struct pci_vdev *vpci_init_vdev(struct acrn_vpci *vpci, struct acrn_vm_pci_dev_config *dev_config, struct pci_vdev *pf_vdev)
```

Comment:
Hi @dongyaozu 

The pf_vdev comment has been added, Need more comments in here?

* @brief Initialize a vdev structure.
 *
 * The function vpci_init_vdev is used to initialize a vdev structure with a PCI device configuration(dev_config)
 * on a specified vPCI bus(vpci). If the function vpci_init_vdev initializes a SRIOV Virtual Function(VF) vdev structure,
 * the parameter pf_vdev is the VF associated Physical Function(PF) vdev structure, otherwise the parameter pf_vdev is NULL.
 * The caller of the function vpci_init_vdev should guarantee execution atomically.
 *
 * @param vpci       Pointer to a vpci data structure
 * @param dev_config Pointer to a dev_config structure
 * @param pf_vdev    Pointer to a SRIOV PF vdev structure